spec/features/order/user_updates_order_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+feature 'Updating an Order', %Q{
+  As a user
+  I want to be able to update an order status and reagent lots
+  So that I can inform the rest of the team of the status of their order
+  } do
+    # Acceptance Criteria:
+    # [] I must see the details of the order
+    # [] I must be able to change the status of the order
+    # [] I must be able to add the lot numbers
+    # [] I must be logged in to update an order
+
+    context "User is signed in and there are existing orders" do
+      before :each do
+        user = FactoryGirl.create(:user)
+        @order = FactoryGirl.create(:order, needed_by: "01/01/2001")
+        FactoryGirl.create(:kit_order, order_id: @order.id)
+        visit new_user_session_path
+
+        fill_in 'Email', with: user.email
+        fill_in 'Password', with: user.password
+
+        click_button 'Log in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.